kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/SolverServicesClientImpl.java

```diff
@@ -100,13 +104,15 @@ public SolverServicesClientImpl(KieServicesConfiguration config, ClassLoader cla
     public ServiceResponse<SolverInstance> updateSolverState(String containerId, String solverId, SolverInstance instance) {
         checkMandatoryParameter( "ContainerID", containerId );
         checkMandatoryParameter( "SolverId", solverId );
+        checkMandatoryParameter( "instance", instance );
```

Comment:
The update method requires the solver state attribute that is inside the solver instance, so it is mandatory.

I agree that having client methods for specific operations would make it easier to use, but as of today, this is the same across drools/jbpm as well. We can introduce the additional client operations as part of the next version.